sdg/IndicatorExportService.py

```diff
@@ -36,6 +47,14 @@ def __get_all_indicator_csv_files(self):
     def __file_is_csv(self, file_name):
         return file_name.endswith(".csv")
 
+    def __file_has_complete_statistical_data(self, file_name):
+        indicator_id = file_name.split('.')[0]
+        if indicator_id not in self.__indicators:
+            raise KeyError("Could not check whether %s is complete." % indicator_id)
+        complete = self.__indicators[indicator_id].is_complete()
+        statistical = self.__indicators[indicator_id].is_statistical()
+        return complete and statistical
```

Comment:
Good thinking separating this into its own function. Maybe this could be done in `Indicator.py` instead though?

Reply:
@SavvasStephanides Do you mean an `is_complete_and_statistical` method in Indicator.py? Personally I would find that redundant, since it already has `is_complete` and `is_statistical`.

Reply:
Yes, I mean have it in `Indicator.py` and pass *that* function here.

Maybe something like `is_suitable_for_export`, or something similar (it needs to be public to be then accessed by the service. This would encapsulate checking for suitability of an indicator to a single function, plus it would be potentially easy to extend when we need to add additional checks in the future.

Reply:
@SavvasStephanides I just updated that method - with the idea of adding to it in the future if we need more limits on what gets exported.